ironfish-cli/src/commands/wallet/notes.ts

```diff
@@ -58,18 +49,24 @@ export class NotesCommand extends IronfishCommand {
           transactionHash: {
             header: 'From Transaction',
           },
+          ...TableCols.asset({ extended: flags.extended }),
+          value: {
+            header: 'Amount',
+            get: (row) => CurrencyUtils.renderIron(row.value),
+            minWidth: 16,
+          },
           isSpent: {
             header: 'Spent',
             get: (row) => {
               if (row.spent === undefined) {
                 return '-'
               } else {
-                return row.spent ? `✔` : `x`
+                return row.spent ? `✔` : ``
```

Comment:
This gets pushed out because value is variable width. Should we just re-order this so it doesn't get shifted around?

Reply:
Reordered so that this appears before `Asset` and `Amount`